README.md

```diff
@@ -2,13 +2,13 @@
 
 ### Welcome!
 
-This repository contains sample projects ready for import into PDSOE and PASOE, meant to illustrate usage of the **Progress Spark Toolkit**. Please note that all demos are for illustrating server-side micro-services only; there are no UI components bundled as of the December 2020 release (v6.0.0).
+This repository contains sample projects ready for import into PDSOE and PASOE, meant to illustrate usage of the **Progress Spark Toolkit**. Please note that all demos are for illustrating server-side micro-services only; there are no UI components bundled as of the August 2021 release (v6.0.1).
 
-**Note:** This release utilizes toolkit release [v6.0.0](https://github.com/progress/Spark-Toolkit/releases/tag/v6.0.0).
+**Note:** This release utilizes toolkit release [v6.0.1](https://github.com/progress/Spark-Toolkit/releases/tag/v6.0.1).
 
 ## Requirements
 
-**OpenEdge 11.7.6** or **OpenEdge 12.2.0** at minimum, **OpenEdge 11.7.8+** or **OpenEdge 12.2.3+** preferred, with the **Spark.pl** library compiled on your appropriate version (11 or 12).
+**OpenEdge 11.7.8** or **OpenEdge 12.2.4** at minimum, **OpenEdge 11.7.10+** or **OpenEdge 12.2.6+** preferred, with the **Spark.pl** library compiled on your appropriate version (11 or 12).
```

Comment:
Make sure the LTS version are the latest - 12.2.8 has been released. Alt say "use the latest 12.2 LTS" or similar and save having to update this on each release.

Reply:
I used the "+" because constantly updating for latest LTS would be an ongoing thing. We NEED 11.7.8 or 1.2.4, but anything higher would be great.

Reply:
Updated the file to contain more specific and up to date version #'s